playwright/e2e/request-baseline.spec.ts

```
@@ -0,0 +1,195 @@
+  if (['script', 'stylesheet', 'font', 'image'].includes(req.resourceType)) return 'Static Asset';
+  return 'Other';
+}
+
+/**
+ * Bundle landing pages to audit.
+ *
+ * Each entry is the first page a user would hit when navigating to that
+ * bundle. Add more bundles here as needed.
+ */
+const LANDING_PAGES = [
+  { name: 'RHEL Insights',      url: '/insights/dashboard' },
```

ESLint

[error] Delete `·····`

(prettier/prettier)